aiosmtpd/docs/smtp.rst

```diff
@@ -121,11 +121,11 @@ SMTP API
    clients when they first connect to the server.  If not given, the system's
    fully-qualified domain name is used.
 
-   *tls_context* and *require_starttls* are related to the ESMTP ``STARTTLS``
-   command for secure connections to the server, based on `RFC 3207`_.
-   *tls_context* is used as the SSL protocol context, and there is no
-   default.  *tls_context* must be given and *require_starttls* must be True
-   for ``STARTTLS`` to be supported.
+   *tls_context* and *require_starttls*.  The ``STARTTLS`` option of ESMTP
+   (and LMTP), defined in `RFC 3207`_, provides for secure connections to the
+   server. For this option to be available, *tls_context* must be supplied,
+   and *require_starttls* must be ``True``.  See :ref:`tls` for a more in
+   depth discussion on enabling ``STARTTLS``.
```

Comment:
`require_starttls=True` is not required in order for the server to advertise STARTTLS in EHLO. However, it is highly recommended to set `require_starttls=True`.